static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import type {Banner} from 'common/modules/ui/bannerPicker';
+import config from 'lib/config';
+import reportError from 'lib/report-error';
+import {onConsentChange} from '@guardian/consent-management-platform';
+import {mountDynamic} from "@guardian/automat-modules";
+
+import {getUserFromApi} from '../../common/modules/identity/api';
+import {isDigitalSubscriber} from "../../common/modules/commercial/user-features";
+
+const brazeSwitch = config.get('switches.brazeSwitch');
+const apiKey = config.get('page.brazeApiKey');
+const brazeVendorId = '5ed8c49c4b8ce4571c7ad801';
```

Comment:
Note this is not the generic IAB vendor ID, but rather a 3rd-party's specific implementation.

Elsewhere we've opted for `SOURCEPOINT_ID`: https://github.com/guardian/frontend/blob/e5ca12069f9d5d91e39d2742d39afbbc2ce0c910/static/src/javascripts/projects/commercial/modules/comscore.js#L17

Reply:
:+1: Thanks @mxdvl (and sorry for the delayed reply - I've been on holiday). Just to clarify, you're talking about using a more suitable name here that makes it clearer we're referring to a 3rd party vendor?